pkgs/applications/misc/moz-phab/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, python3Packages, pkgs }:
```

Comment:
```suggestion
{ lib, python3Packages}:
```
Please don't pass in pkgs as it is not very performant to override it.

Reply:
should I also not pass `python3Packages` for the same reason?